mockserver_test.go

```diff
@@ -8,6 +8,8 @@ import (
 	"sync"
 	"testing"
 
+	"golang.org/x/crypto/ssh"
+
```

Comment:
No need to split imports.
```suggestion
	"golang.org/x/crypto/ssh"
```